### DISCUSSION TITLE
Error Tasks LAb 04.02

### DISCUSSION THREAD

**Discussion by esn8der** [rramosp/ai4eng.v1#307] [04 - GESTIÓN DE DATOS]
He encontrado varios problemas al momento de enviar mis soluciones de este laboratorio y algunos compañeros que asistieron a la ultima clase hablaron de que había un error con alguna librería y quería confirmar si ese laboratorio se debe hacer o no.

Gracias

**Comment by Felipe-RA** [comment]
Hola  @esn8der 

> Hola a tod@s, 
>
>como muchos de vosotros habéis experimentado >surgió un error en algunos de los calificadores >automáticos del módulo 4. Esto fue debido a que >pandas cambió una semántica de acceso a >dataframe introduciendo una incompatibilidad >con código de versiones anteriores. 
>
>algunos correctores usaban esa sintaxis y por eso fallaban.
>
>los hemos corregido a ahora deberíais de poder >someter vuestras soluciones a los laboratorios del módulo 4.
>
>por esto, y para que tengáis el tiempo adecuado >para trabajar sobre los mismo, vamos a mover la >fecha de entrega de los laboratorios de ese >módulo 4 al 14/abril. La fecha de entrega de los >laboratorios del módulo 3 queda como estaba, >para este próximo 7 abril.
>
>Con esto, haremos el quiz de ambos laboratorios >el próximo jueves 18/abril.
>
>gracias por vuestra comprensión
>saludos
>raúl
